example/example.dart

```diff
@@ -37,6 +35,9 @@ class _MyHomePageState extends State<MyHomePage> {
               onChanged: (bool state) {
                 print('turned ${(state) ? 'on' : 'off'}');
               },
+              onDoubleTap: () {},
```

Comment:
I don't know if this could be the best approach in case of widget customization (in case anyone need to set these parameters) this will figure as a TODO for next releases.

Reply:
All events except onChange will remain as optional, null check on function invocation is used for newer flutter version compatibility.